src/main/antlr/Graphql.g4

```diff
@@ -145,9 +145,13 @@ inputObjectTypeDefinition
 
 scalarTypeDefinition : description? SCALAR name directives?;
 
-objectTypeDefinition : description? TYPE name implementsInterfaces? directives? '{' fieldDefinition+ '}';
+objectTypeDefinition : description? TYPE name implementsInterfaces? directives? fieldsDefinition?;
 
-implementsInterfaces : IMPLEMENTS typeName+;
+implementsInterfaces :
+    IMPLEMENTS '&'? typeName+ |
+    implementsInterfaces '&' typeName ;
```

Comment:
Trying to understand this part. It says "implements keyword, maybe & and 1 or more type names or ... this recursively again?". Does it mean "X implements & A & B" is legal? How about "X implements A implements B"?

Reply:
Yeah it seems like the spec grammar as written is weird or wrong

I think the intention is for

    type Foo implements Bar & Baz & Buz

not the weird combos this would allow - but do I follow spec or commons sense?

Reply:
It seems identical to the reference implementation, and that's the least surprising possibility, so I'd call it correct.